### ISSUE TITLE
Class "October\Rain\Router\UrlGenerator" not found

### ISSUE THREAD

**Issue by meysammahfouzi** [cydrickn/octobercms-backendskin#10]
After installing the plugin, I get the following error:

```
Class "October\Rain\Router\UrlGenerator" not found
```

It seems that the `UrlGenerator` does not exist in October anymore and you should update your plugin.

**Comment by daftspunk** [comment]
Update this file: https://github.com/cydrickn/octobercms-backendskin/blob/594e09f341f9912005ca8f4fb1c3dfc25322da36/router/UrlGenerator.php#L6

From:
```
use October\Rain\Router\UrlGenerator as Base;
```

To 
```
use Illuminate\Routing\UrlGenerator as Base;
```

**Comment by cydrickn** [comment]
This will be included in release 1.2.1
